app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -708,8 +713,12 @@ private void focusOnTouch(MotionEvent event) {
 
         //Calculate the Focus Area
         private Rect calculateFocusArea(float x, float y) {
-            int left = clamp(Float.valueOf((x / mSurfaceView.getWidth()) * 2000 - 1000).intValue(), FOCUS_AREA_SIZE);
-            int top = clamp(Float.valueOf((y / mSurfaceView.getHeight()) * 2000 - 1000).intValue(), FOCUS_AREA_SIZE);
+			//getting screen size
+			screen_width = Utils.getScreenWidth(getContext());
+			screen_height = Utils.getScreenHeight(getContext());
```

Comment:
OK fine.........if that matters I will change it. 
By the way, it is indentation, not intendation.